Cargo.toml

```diff
@@ -131,14 +135,14 @@ uuid = { version = "0.7", features = ["serde", "v4"], optional = true }
 exitcode = "1.1.2"
 snafu = { version = "0.6", features = ["futures-01", "futures"] }
 url = "2.1.1"
-base64 = { version = "0.10.1", optional = true }
+base64 = { version = "0.12.3", optional = true }
 bollard = { version = "0.7.1", default-features = false, features = ["tls"], optional = true }
 listenfd = { version = "0.3.3", optional = true }
 inventory = "0.1"
 maxminddb = { version = "0.14.0", optional = true }
 strip-ansi-escapes = { version = "0.1.0"}
 colored = "1.9"
-warp = { version = "0.2.4", default-features = false, optional = true }
```

Comment:
Perhaps we can only enable the default features we actually need? (unless we need them all?)

Reply:
Yeah, that sounds good. I removed `default-features = false` temporarily to instrument Warp in the API, with the intention of circling back and testing for what we actually need. Thanks for the reminder.

Reply:
FWIW, adding `default-features = false` back to the warp definition made no difference to `cargo build`-- it compiled the cached version just the same. Since we're not also specifying a definitive `features = []` here, I'm not clear on if there's a distinction.